docs/src/_includes/layouts/base.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="{{ config.lang }}" class="no-js">
+<html lang="{{ config.lang }}" class="no-js" id="scroll_to_top">
```

Comment:
lets rename `scroll_to_top` with something else? like page-start or some other that makes sense